modules/machineset-creating-azure-ephemeral-os.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+
+[id="machineset-creating-azure-ephemeral-os_{context}"]
+= Creating machines on Ephemeral OS disks by using machine sets
+
+You can launch machines on Ephemeral OS disks on Azure by editing your machine set YAML file.
+
+.Procedure
+
+. Add the following to the `providerSpec` field:
```

Comment:
Should we put a step to edit the machine set YAML to the procedure, in case folks skip the intro paragraph? 
Is there a specific machine set or whichever  one moves the nodes you want on Ephemeral OS disks?